README.md

```diff
@@ -30,29 +30,23 @@ inference.
     ```
     sudo apt-get install libturbojpeg
     sudo apt-get install python3-tk
-    virtualenv ./toolbox_env -p python3
-    source ./toolbox_env/bin/activate
-    pip install -r requirements.txt
     ```
 
-3. Build COCO API
+3. Create virtual environment
 
     ```
-    cd external/cocoapi
-    2to3 . -w
-    cd PythonAPI
-    python setup.py build_ext --inplace
+    sh tools/init_venv.sh
     ```
 
-4. Update PYTHONPATH environment variable
+4. Start to work
```

Comment:
You have changed the project structure, so tests have a general structure for the whole project. Maybe step 5 should be "Run tests".

Reply:
test section was added